### ISSUE TITLE
black screen after opencore menu

### ISSUE THREAD

**Issue by Not-Saii** [poopsicles/hackbook#2]
So, I got a hp probook 640 G1 as well, and trying to get this working.

I clone the repo, drag the config.plist into GenSMBIOS thing and generate the SMBIOS using "MacBookPro11,5"
When booting to the usb it doesn't show it in the opencore menu, after a few refreshes and replugging the usb it shows up

Got further with [this](https://github.com/shreyas-shriyan/Hp-Probook-640G1-Hackintosh-OpenCore) config, but it would get stuck on the screen with apple logo and loading bar, it wouldn't move.

**Comment by Not-Saii** [comment]
UPDATE
after using a different drive managed to get to the installer, after the install "finishes" it just goes to a black screen.

Also just recently noticed that I'm using a 640 not a 650 but managed to get further with this than that other EFI

**Comment by dibenzepin** [comment]
well, you mentioned on Reddit that you managed to get it to boot, so maybe make a PR to the repo you linked?

**Comment by Not-Saii** [comment]
FINAL UPDATE

Turns out I'm just impatient, and it needs to be manually rebooted a few times to get it to work, just boot back into the installer until it finishes. Also an empty drive helps and a normal memory amount (had 12gigs but after installing OS put them back and it works). Only things that I noticed that don't work are 3.5 jacks mic input and audio over Bluetooth to my airpods but ear detection and auto pause works so idk. In conclusion, be patient, it took like 2 hours to install and around 3 reboots, after that do [this](https://dortania.github.io/OpenCore-Post-Install/universal/oc2hdd.html), just mount the EFI of the drive you installed OS on and copy BOOT and OC folders into the EFI folder.
